dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -1098,6 +1128,162 @@ async fn cmd_db_dns_names(
     Ok(())
 }
 
+async fn cmd_db_eips(
+    opctx: &OpContext,
+    datastore: &DataStore,
+    limit: NonZeroU32,
+    verbose: bool,
+) -> Result<(), anyhow::Error> {
+    /*
+    let ips = datastore
+        .lookup_external_ips(&opctx)
+        .await
+        .context("listing external ips")?;
+    */
```

Comment:
nit: probably want to remove this
I can't tell if this was here before because I can't access the incremental review.  (Maybe a force push?)